block_server.py

```diff
@@ -664,15 +657,17 @@ def consume_write_queue(self):
             self.load_config, ("configname",), "LOADING_CONFIG")
         """
         while True:
-            while len(self.write_queue) > 0:
-                with self.write_lock:
+            with self.write_lock:
+                if self._filewatcher is not None: self._filewatcher.pause()
```

Comment:
I think the original reason I did it this way was that I was concerned about constantly pausing and resuming the filewatcher, but the suggested way is definitely better. Changed 	52c0827